electron/app/containers/Setup.tsx

```diff
@@ -1,47 +1,137 @@
-import React, { createRef, useState } from "react";
-import { Container, Header, Divider, Segment, Menu } from "semantic-ui-react";
+import React, { useState, useContext } from "react";
 import { Redirect } from "react-router-dom";
+import styled, { ThemeContext } from "styled-components";
+import { animated, useSpring } from "react-spring";
 
 import routes from "../constants/routes.json";
 import connect from "../utils/connect";
-import CodeBlock from "../components/CodeBlock";
-import PortForm from "../components/PortForm";
-import RemoteInstructions from "../components/RemoteInstructions";
-import LocalInstructions from "../components/LocalInstructions";
+import localSnippet from "../snippets/local.py";
+import bashSnippet from "../snippets/remote.bash";
+import remoteSnippet from "../snippets/remote.py";
+
+const SectionTitle = styled.div`
+  font-size: 2rem;
+  line-height: 3rem;
+  color: ${({ theme }) => theme.font};
+  font-weight: bold;
+`;
+
+const Text = styled.p`
+  font-size: 1rem;
+  line-height: 1.5rem;
+  margin: 0;
+  padding: 0;
+  color: ${({ theme }) => theme.fontDark};
+`;
+
+const Code = styled.pre`
+  padding: 2rem;
+  background-color: ${({ theme }) => theme.backgroundDark};
+  border: 1px solid ${({ theme }) => theme.backgroundDarkBorder};
+  color: ${({ theme }) => theme.secondary};
```

Comment:
I'm a bit concerned that this could be harder to read on a black background than it was before. Bumping up the brightness (e.g. from 53% to 73%) is a bit better, but I think a light grey would also be fine, e.g. `grey68`.